doc/source/tracking/icepyx_pubs.bib

```diff
@@ -36,15 +36,15 @@ @misc{2022_IS2-HW-tutorials
 }
 
 @misc{js2021agu,
-author = {Scheick, J and Bisson, K and Li, T and Leong, WJ and Arendt, A},
+author = {Scheick, J. and Bisson, K. and Li, T. and Leong, W.J. and Arendt, A.},
```

Comment:
> otherwise there's J. Scheick and Jessica Scheick in the PDF. BibTeX is not so smart to know that they're the same person 🙂

Agreed, but it SHOULD be smart enough to consistently not use first names even where they're present (which it managed to do in ALL other cases where a full first name is provided). I'd been hoping adding the period would help the BibTeX parser, but apparently a manual solution is still required...

Reply:
Yeah, if only if knows... The issue is that the first name will need to be printed in the in-text citation if say, you have Ben Smith and Brian Smith. Here, BibTeX doesn't know if J. Scheick is Joan Scheick or Jessica Scheick or someone else, so it errs on the side of caution and treats them separately.

Source: personal experience having to do this on the last day before handing in my PhD thesis - https://github.com/weiji14/phdthesis/commit/ae4a8abce4fd91c6baedc4c51298d19b897c11aa#diff-f51b46ebf5edcd7495d25563b61b422ae8959770cf858b76686be8c8ae034cf7